app/views/form/_radio_question.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= f.govuk_radio_buttons_fieldset question_key,
-    legend: { text: question["header"], size: "l" },
+    legend: { text: question["header"].html_safe, size: "l" },
```

Comment:
Should we add this to the other question type partials so that they're capable of rendering this too?

Reply:
great idea! 😅